lib/apiservers/engine/backends/container.go

```diff
@@ -317,6 +317,11 @@ func (c *Container) ContainerRename(oldName, newName string) error {
 func (c *Container) ContainerResize(name string, height, width int) error {
 	defer trace.End(trace.Begin("ContainerResize"))
 
+	vc := cache.ContainerCache().GetContainer(name)
+	if vc == nil {
+		return derr.NewRequestNotFoundError(fmt.Errorf("No such container: %s", name))
```

Comment:
Do not return err here, as you check its existence later via PL. Instead just do
```
vc := cache.ContainerCache().GetContainer(name)
if vc != nil {
name = vic
}
```